src/transformers/baseSourceMapTransformer.ts

```diff
@@ -40,6 +40,7 @@ export class BaseSourceMapTransformer {
     private _requestSeqToSetBreakpointsArgs: Map<number, ISavedSetBreakpointsArgs>;
     private _allRuntimeScriptPaths: Set<string>;
     private _authoredPathsToMappedBPs: Map<string, DebugProtocol.SourceBreakpoint[]>;
+    private _authoredPathsToClientBreakpointIds: Map<string, number[]>;
```

Comment:
I hate that we have to keep track of the breakpoint IDs in here. But we're already keeping track of the breakpoints anyway. It's not great. Let me think about this a little more.

Reply:
Okay let me know!

Reply:
@roblourens: Did you get a chance to think about this yet?